core_lib/src/interface/scribblearea.cpp

```diff
@@ -398,26 +398,29 @@ void ScribbleArea::wheelEvent(QWheelEvent* event)
 
     const QPoint pixels = event->pixelDelta();
     const QPoint angle = event->angleDelta();
-    //qDebug() <<"angle"<<angle<<"pixels"<<pixels;
-
+    #if (QT_VERSION >= QT_VERSION_CHECK(5, 14, 0))
+    const QPointF offset = mEditor->view()->mapScreenToCanvas(event->position());
+    #else
+    // This is actually not the same as the above, hence why we use it only if necessary
+    const QPointF offset = mEditor->view()->mapScreenToCanvas(mapFromGlobal(QCursor::pos()));
+    #endif
+
```

Comment:
The comment mentions that these doesn't give the same outcome, but as far as I can see they do... at least using my trackpad scroll? 

is there any particular reason why you use ```mapFromGlobal(QCursor::pos()))``` 
instead of event->posF() or event->pos()?

Reply:
THANK YOU event->posF() is exactly what I actually wanted but I just couldn’t find it because it is deprecated and hidden in the documentation 🤦. I’ve now removed the version check altogether like we did with the similarly deprecated byteCount() in MovieExporter.